setup.py

```diff
@@ -7,9 +7,8 @@
     version=0.1,
     author="Matt Favoino, Allan Liebold",
     licence="MIT",
-    py_modules=['linked-list', 'stack', 'dll', 'que_', 'deque',
-                'priorityq', 'binheap', 'graph', 'graph_2', 'graph_3',
-                'bst'],
+    py_modules=['linked-list', 'dll', 'stack', 'queue'],
+    py_modules=['linked-list', 'stack', 'dll', 'que_'],
```

Comment:
Why is this spread across two lines instead of just having a list once? Having repeated keyword arguments breaks your install.